### PR TITLE
Changed my name in index.html

### PR DESCRIPTION
**Pull request by JMStewart00**
- changed from Josh to Joshua
- updated xyz
- did something else
- as many as you want

### CODE REVIEW

**Comment by JMStewart00** [comment]
@anchormansam

File: index.html [hunk 1]
```diff
@@ -76,7 +76,7 @@ <h4 class="mt-5 mb-0 pb-0">Full Stack Web Developer</h4><br>
 
 			<div class="row mb-5 p-2">
 
-				<h3 class="introTitle text-center px-3 text-white">Hello, I'm Josh. I've always been drawn to the overlap between <strong>design and development.</strong> My skills are broad: from ux to design, front end to back end development. I enjoy each aspect, and love building sites & mobile apps from start to finish with whatever tools you wish.</h3>
+				<h3 class="introTitle text-center px-3 text-white">Hello, I'm Joshua. I've always been drawn to the overlap between <strong>design and development.</strong> My skills are broad: from ux to design, front end to back end development. I enjoy each aspect, and love building sites & mobile apps from start to finish with whatever tools you wish.</h3>
```
Comment (Wildercat) [inline index.html:79]: Who's Joshua?

**Comment by Leozas** [comment]
Theres to much text. Learn to minify code.

File: index.html (hunk 1 above)
Comment (eBork1) [reply index.html:79]: :)

**Pull request closed without merging**